scripts/globals/wsquest.lua

```diff
@@ -0,0 +1,562 @@
+require('scripts/globals/keyitems')
+require('scripts/globals/quests')
+require('scripts/globals/status')
+
+dsp = dsp or {}
+dsp.wsquest =
+{
+    --------------------------------------
+    -- Quest: The Walls of Your Mind
+    --   NPC: Oggbi (Port Bastok)
+    --    WS: Asuran Fists (Hand-to-Hand)
+    --------------------------------------
+    ['theWallsOfYourMind'] =
```

Comment:
Just thinking out loud here. I am wondering if this would be a bit more intuitive to rename these to either the combat skill itself, or the weapon skill it unlocks, 

`dsp.wsquest.theWallsOfYourMind` really isn't very obvious what it's for without some digging, where something like `dsp.wsquest.hand_to_hand` or `dsp.wsquest.asuran_fists` is pretty obvious.